src/main/groovy/com/getkeepsafe/dexcount/System.groovy

```diff
@@ -0,0 +1,38 @@
+package com.getkeepsafe.dexcount
```

Comment:
Let's keep the original package names, ideally the original file content as well.

On the latter point, what did we have to change here, besides the package name?